scripts/actions_local_runner.py

```diff
@@ -0,0 +1,138 @@
+#!/bin/python3
```

Comment:
i dont know if there's a rule on what gets put into `.github/scripts` and what in `scripts/`

Reply:
Well according to `scripts/README.md`:

```
This directory contains the useful tools.

...
```

but then in the `tools/README.md`:

```
This folder contains a number of scripts which are used as
...
```

so it looks like it's made to confuse

Reply:
lol yep, as noted in #52688